src/components/teampage_infoblock.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import propTypes from "prop-types";
+import styled from "styled-components";
+import Img from "gatsby-image";
+
+const Centered = styled.h5`
+  text-align: center;
+  margin-top: 10px;
+`;
+
+const Centered2 = styled.h6`
+  text-align: center;
+  margin-top: 10px;
+`;
+
+const Centered3 = styled.h6`
+text-align: center;
+margin-bottom: 30px
+flexGrow: 0;
+flexBasis: auto;
+flexShrink: 1;
+flex-wrap: wrap;
+flexDirection: row;
+color: var(--MHP-purple);
+`;
+
+const colour = {
+  color: "#4831d4",
+};
```

Comment:
This should be done using the global CSS (`global.css`).
 eg. add `color: var(--MHP-purple)` to a styled component to add the colour.

Reply:
I tried to do this (looks like in line 24 that I forgot to remove) but it doesn't change the colour for some reason. Have any idea why?